packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/index.js

```diff
@@ -142,7 +159,7 @@ const Component = ({
                 )}
               </ActionStack>
             }
-            title={friendlyName}
+            title={`${friendlyName} - ${mainValue}`}
```

Comment:
I think we need to truncate the text, so that long values don't break the layout. This can either be achieved by using the `Typography` component with an `ellipsis` prop or by cutting of the text at a max length here.

Reply:
I did not see ellipsis option for title in AccordionToggle, so I have turcated the mainValue when longer then 50.

Reply:
I have tried the Typography ellipsis in here, the title prop validation did not expect JSXElement only String (it can be  changed).

```jsx
<AccordionToggle title={<Typography ellipsis>{`${friendlyName} - ${mainValue}`}</Typography>}
```

But the ellipsis not working without width or max-width I think.

```js
Warning: Failed prop type: Invalid prop `title` of type `object` supplied to `AccordionToggle`, expected `string`.
    at AccordionToggle (webpack-internal:///./src/Accordion/AccordionToggle.js:62:21)
```

![strapi-dzone-ellipsis-wo-width](https://user-images.githubusercontent.com/912200/156318598-16699f78-c6ba-4ef7-bad8-14652e0c494a.jpg)

Reply:
Oh I see. I'll talk to a designer and try to find out if they have an opinion on that.

Reply:
Ok, could I do something, to help the PR to move forward?

Reply:
Seconding this @gu-stav. This PR solves a very annoying editor UX issue, I really hope it can move forward!

Reply:
@godzzo @JesperWe My apologies - I had too many other tasks on my plate the last week. I think wrapping the value similar to the `AccordionToggle` works well for our case.

Reply:
No problem :) Ok, then I will remove the string length limit condition.